plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -172,269 +178,264 @@
-                description: Mode of power assignment for the 2.4GHz radio interface. Accepts "Global" or "Custom".(eg. "Custom")
-                type: str
-                required: False
-              power_level:
-                description: Custom power level configured for the 2.4GHz radio interface. (eg. 3)
-                type: int
-                required: False
-          5ghz_radio:
-            description: Configuration options for the 5GHz radio interface.
-            type: dict
+          antenna_name:
+            description: Name or type of antenna used for the 2.4GHz radio interface.(eg. "other")
+            type: str
             required: False
-            suboptions:
-              admin_status:
-                description: Administrative status for the 5GHz radio interface.(eg. "Enabled")
-                type: str
-                required: False
-              antenna_name:
-                description: Name or type of antenna used for the 5GHz radio interface.(eg. other)
-                type: str
-                required: False
-              antenna_gain:
-                description: Antenna gain value in decibels (dB) for the 5GHz radio interface.(eg. 5)
-                type: int
-                required: False
-              radio_role_assignment:
-                description: Role assignment mode for the 5GHz radio interface. Accepts "Auto", "Client-serving", or "Monitor".(eg. Auto)
-                type: str
-                required: False
-              cable_loss:
-                description: Cable loss in dB for the 5GHz radio interface. (eg. 80)
-                type: int
-                required: False
-              antenna_cable_name:
-                description: Name or type of antenna cable used for the 5GHz radio interface.(eg. other)
-                type: str
-                required: False
-              channel_assignment_mode:
-                description: Mode of channel assignment for the 5GHz radio interface. Accepts "Global" or "Custom".(eg. "Custom")
-                type: str
-                required: False
-              channel_number:
-                description: Custom channel number configured for the 5GHz radio interface. (eg. 36)
-                type: int
-                required: False
-              power_assignment_mode:
-                description: Mode of power assignment for the 5GHz radio interface. Accepts "Global" or "Custom".(eg. "Custom")
-                type: str
-                required: False
-              power_level:
-                description: Custom power level configured for the 5GHz radio interface. (eg. 3)
-                type: int
-                required: False
-          6ghz_radio:
-            description: Configuration options for the 6GHz radio interface.
-            type: dict
+          antenna_gain:
+            description: Antenna gain value in decibels (dB) for the 2.4GHz radio interface.(eg. 4)
```

Comment:
Specifies the antenna gain value in decibels (dB) for the 2.4GHz radio interface. For example, 4.

Can we use "For example, value" instead of (eg. value) in all the places?